src/server.js

```diff
@@ -39,6 +43,19 @@ server.route({
 });
 
 /**
+ * Serve static bundled client.js, needed for production server
+ */
+server.route({
+  method: 'GET',
+  path: '/dist/client.js',
```

Comment:
is there not a static assets setting in hapi, similar to `express.static` - or do you need to define a route for every file? 😕